CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # v 3.5.0 (?)
 Changes in this release:
 - Make sure that the orchestrators started by pytest-inmanta-lsm log their output to `/var/log/inmanta/server.log` instead of stdout for rc containers.
+- Fix compatibility with updated ServiceInstance model -> new fields:
+  - `desired_state_version`
+  - `transfer_context`
+
+  Refer to inmanta-lsm for more details.
```

Comment:
People using the library (which is open source and used by clients) can't refer to inmanta-lsm as it is closed source